67_RayQueryGeometry/app_resources/render.comp.hlsl

```diff
@@ -0,0 +1,160 @@
+#include "common.hlsl"
+
+#include "nbl/builtin/hlsl/spirv_intrinsics/core.hlsl"
+
+struct SPushConstants
+{
+    SGeomBDA geom[OT_COUNT];
+};
```

Comment:
fun fact, push constants CANNOT BE DYNAMICALLY INDEXED (@kpentaris got cut on it once)

You want to have the `SGeomBDA` in a IGPUBuffer somewhere (maybe call it `SGeomInfo), also put the vertex offsets and strides in there or at least `SVertexInfo`.

then you want a BDA address to the `SGeomBDA` in the push constants and `vk::RawBufferLoad` them

Reply:
and the camera data you want in the push constants